File: Python/grading-students.py
def gradingStudents(grades):
    fingrade = []
    for i in range(len(grades)):
        if(grades[i]>=38 and grades[i]%5>=3):
            val = (grades[i]+5)-(grades[i]%5)
        else:
            val = grades[i]
        fingrade.append(val)
    return fingrade

File: Java/grading-students.java
class Result {
    public static List<Integer> gradingStudents(List<Integer> grades) {
        List<Integer> fingrade = new ArrayList<>();
        for (int i = 0; i < grades.size(); i++) {
            int grade = grades.get(i);
            if(grade >=38 && grade%5>=3){
                grade = grade + 5 - grade % 5;
            }
            fingrade.add(grade);
        }
        return fingrade;
    }
}